openadapt/config.py

```diff
@@ -41,7 +41,8 @@
     "SCRUB_CHAR": "*",
     "SCRUB_LANGUAGE": "en",
     # TODO support lists in getenv_fallback
-    "SCRUB_FILL_COLOR": (255, 0, 0),
+    "TEST_SCRUB_FILL_COLOR": (255, 0, 0),
+    "SCRUB_FILL_COLOR": (255,),
```

Comment:
Can you please clarify what this is for?

Reply:
For running pytest test_scrub.py function: , it will need a full tuple with all the 3 values of RGB. eg. (255, 0, 0) for comparison. this is because the tests do not pass when we have (255,) as it does not recognize as the Red colour.

Reply:
And if we have (255,0,0) as config.SCRUB_FILL_COLOR then we get an error as described in this PR description. For which we have to change the the tuple to (255,)

Reply:
To sum up, to pass the pytest as well as run the visualize without getting the error mentioned in this PR description, 
I added both to config.